packages/ui/src/components/AuthorRow.tsx

```diff
@@ -50,13 +55,25 @@ export default function AuthorRow({ onPress, ...props }: AuthorRowProps) {
   return (
     <>
       {props.detailView ? (
-        <DetailViewAuthorRow {...props} onPress={handlePress} />
+        <DetailViewAuthorRow
+          {...props}
+          onPress={nonPressable ? undefined : handlePress}
```

Comment:
Would just setting a `disabled` prop on `AuthorRow` allow us to skip this prop + the duplication here? Also, depending on what you're trying to achieve `pointerEvents="none" ` would probably work.